src/components/Profile/Profile.css

```diff
@@ -43,7 +43,7 @@
   display: flex;
   gap: 1rem;
   height: 35px;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
This doesn't fix the issue. Please the issue properly.

Reply:
I need little more clarification on this.
When I run the project on my local system, I can see the scroll is working.

Reply:
can you share video recording of that?